### PR TITLE
Remove the fg CustomResourceValidationExpressions

### PR DESCRIPTION
**Pull request by cici37**
Doc for merged change:  https://github.com/kubernetes/kubernetes/pull/126136

### CODE REVIEW

**Comment by Shubham82** [comment]
Thanks @cici37
/lgtm

**Review by kundan2707 — COMMENTED** [review]
Looks good as feature has been reomoved.
thanks

File: content/en/docs/reference/command-line-tools-reference/feature-gates/custom-resource-validation-expressions.md [hunk 1]
```diff
@@ -16,7 +16,10 @@ stages:
     toVersion: "1.28" 
   - stage: stable
     defaultValue: true
-    fromVersion: "1.29"  
+    fromVersion: "1.29"
+    toVersion: "1.31"
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/custom-resource-validation-expressions.md:20]: This should be 1.30?

**Comment by cici37** [comment]
/assign @kubernetes/sig-docs-en-owners 
for approval. Thanks

File: content/en/docs/reference/command-line-tools-reference/feature-gates/custom-resource-validation-expressions.md (hunk 1 above)
Comment (cici37) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/custom-resource-validation-expressions.md]: ops. Updated. Thanks

**Pull request merged**